### ISSUE TITLE
Cannot find "object frame" in RVIZ plugin.

### ISSUE THREAD

**Issue by ZZWang21** [ros-planning/moveit_calibration#114]
Hi, dear experts. I have installed the handeye calibration plugin in RVIZ. However, when I tried to make the calibration. I cannot find anything in "object frame". I tried to reinstalled Moveit but I did not find any "handeye calibration" plugin in the Moveit Setup Assistant. What could be the problem? Thank you for your help. @felixvd  @RoboticsYY

**Comment by RoboticsYY** [comment]
Hi @ZZWang21 moveit_calibration is already a standalone tool, not part of the moveit_setup_assistant. Please follow the tutorial https://ros-planning.github.io/moveit_tutorials/doc/hand_eye_calibration/hand_eye_calibration_tutorial.html to setup and use this tool.

**Comment by ZZWang21** [comment]
> Hi @ZZWang21 moveit_calibration is already a standalone tool, not part of the moveit_setup_assistant. Please follow the tutorial https://ros-planning.github.io/moveit_tutorials/doc/hand_eye_calibration/hand_eye_calibration_tutorial.html to setup and use this tool.

Thank you @RoboticsYY  for the reply. I build the package following the instruction. I planned to make the hand-eye calibration in the simulation environment, is this possible? Or, is it only for the real robot? The "object frame" can only show up when the target is detected? So, I think I should input an image in RVIZ environment? I do not know if I understand it right. Thank you very much for your reply.

**Comment by RoboticsYY** [comment]
Yes, it is possible to run with robot arm and moveit in simulation, but you have to get some camera messages published or connect to some camera. Yes, the "object frame" can only show up when the target is detected. In the Rviz GUI, you can select the topic name of the camera image message. Then the target detection plugin will detect the aruco marker target pose.

**Comment by ZZWang21** [comment]
> Yes, it is possible to run with robot arm and moveit in simulation, but you have to get some camera messages published or connect to some camera. Yes, the "object frame" can only show up when the target is detected. In the Rviz GUI, you can select the topic name of the camera image message. Then the target detection plugin will detect the aruco marker target pose.

Thank you very much. I have camera topic in my model. But currently, I did not put the target image in simulation. I will put the image into the simulation environment later to check. Thank you very much for your reply.

**Comment by pwyq** [comment]
@RoboticsYY Hi, do you happen to have a working example of the package? I also stuck on the unavailable "object frame" part.

I tried to build on an ABB robot arm and adding a camera in the model. The RViz image topic shows what Gazebo's camera sees, but the camera is falling in Gazebo (which I guess that's why my "object frame" is missing).
